packages/components/NativeRadioButton/src/NativeRadioButton.tsx

```diff
@@ -0,0 +1,68 @@
+/** @jsx withSlots */
+import { ViewProps } from 'react-native';
+import { compose, buildProps, mergeProps, UseSlots, LayoutTokens } from '@fluentui-react-native/framework';
+import { ensureNativeComponent } from '@fluentui-react-native/component-cache';
+import { withSlots } from '@uifabricshared/foundation-composable';
```

Comment:
I think you can import this from the new framework and not take a dependency on the deprecated one

Reply:
Yeah, like we do in Shimmer.tsx
```
import { ..., withSlots, ... } from '@fluentui-react-native/framework';
```

Reply:
I'm not sure why it can't find `withSlots` from @fluentui-react-native/framework, that only happens in non-experimental components.